src/reducers/PagesReducer.js

```diff
@@ -22,10 +21,6 @@ export const PagesReducer = (state = {}, action) => {
       return { ...state, currentRoute: prevRouteName };
     }
 
-    case FINALISE_ACTIONS.CONFIRM_FINALISE: {
```

Comment:
just as above - flinging off a refresh data action rather than using this action now